__tests__/app.test.js

```diff
@@ -109,4 +109,45 @@ describe('3. GET /api/articles/:article_id', () => {
         expect(body.msg).toBe('invalid article id')
       });
   });
-});
\ No newline at end of file
+});
+
+describe('4. GET /api/articles/:article_id/comments', () => {
+  test('status:200, responds with an array of comment objects for a given article', () => {
+    return request(app)
+      .get('/api/articles/1/comments')
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+        expect(comments).toBeInstanceOf(Array);
+        expect(comments).toBeSortedBy("created_at", {descending: true})
+        comments.forEach((comment) => {
+          expect(comment).toEqual(
+            expect.objectContaining({
+                comment_id: expect.any(Number),
+                votes: expect.any(Number),
+                author: expect.any(String),
+                body: expect.any(String),
+                created_at: expect.any(String),
+            })
+          );
+        });
+      });
+  });
```

Comment:
One more test worth doing is when the article DOES exist but it has no comments. Just because an article has no comments doesn't mean it doesn't exist. How does your code handle such a situation?

Apart from this one missing test, it's a great testing suite here.